core/codecArrayBuffer.js

```diff
@@ -88,7 +88,41 @@ sjcl.codec.arrayBuffer = {
     return out;
   },
 
+  /** Add padding with zero bytes if needed.
+   * This is the same padding as applied in fromBits where
+   * bitArray gets converted to ArrayBuffer.*/
+  padBuffer: function (buffer, padding, padding_count) {
+    var ol, out, i;
+    padding = padding==undefined  ? true : padding
+    padding_count = padding_count || 16
 
+    if (buffer.byteLength === 0) {
+      return new ArrayBuffer(0);
+    }
+
+    ol = buffer.byteLength;
+    if (padding && ol%padding_count !== 0){
+      ol += padding_count - (ol%padding_count);
+    }
+
+    var out = new Uint8Array(ol);
+      out.set(new Uint8Array(buffer), 0);
+      return out.buffer;
+  },
+
+  /** Convert decrypted bitArray into buffer with original length.
+   * When plaintext is obtained by decryption, it is in the form of bitArray.
+   * If this bitArray would be converted using fromBits, there would be some
+   * zeros appended at the end (compared to the original ArrayBuffer prior encryption). **/
+  toBuffer: function (bitArray) {
```

Comment:
Yes, somehow I missed it. It is fixed now - function fromBits is used a test and toBuffer is removed.
However, I don't know what to do to fix a build failure (now padBuffer is not found).